src/App.js

```diff
@@ -7,23 +7,20 @@ class App extends Component {
     super(props)
     this.state = {
       squares: [null, null, null, null, null, null, null, null, null],
-      turn: 0,
-      lastTurn: null
+      player: 1,
```

Comment:
renamed turn to player and changed value to 1. removed lastTurn as it was unused